components/CalEPD/models/dke/depg750bn.cpp

```diff
@@ -0,0 +1,352 @@
+
+#include "dke/depg750bn.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include "esp_log.h"
+#include "freertos/task.h"
+#include <stdint.h>
+#include <stdbool.h>
+#include <inttypes.h>
+
+#define ENABLE		1
```

Comment:
Last small thing, cleanup: are this ENABLE / DISABLE even used?

Reply:
I usually prefer using ENABLE / DISABLE instead of  1 and 0s 
I have removed them no worries.